plasmapy/particles/tests/test_exceptions.py

```diff
@@ -940,7 +946,7 @@ def test_named_tests_for_exceptions(tested_object, args, kwargs, expected_except
     [particle_mass, ["fe-56 1+"], {}, pytest.raises(InvalidParticleError)],
     [is_stable, ["hydrogen-444444"], {}, pytest.raises(InvalidParticleError)],
     [is_stable, ["hydrogen", 0], {}, pytest.raises(InvalidParticleError)],
-    [is_stable, [""], {}, pytest.raises(InvalidParticleError)],
+    [is_stable, [""], {}, pytest.raises(ParticleError)],
```

Comment:
Just curious - why did this change if both ParticleError and InvalidParticleError still exists? Seems like `""` should be an invalid particle?

Reply:
That was a good question!  I decided to dig into it.

This is the end of the traceback for the error I got when it was expected a `InvalidParticleError`:

```Python
plasmapy/particles/decorators.py:531: in process_argument
    self.verify_particle_categorization(particle)
plasmapy/particles/decorators.py:397: in verify_particle_categorization
    raise ParticleError(errmsg)
E   plasmapy.particles.exceptions.ParticleError: The particle ParticleList([]) does not meet the required classification criteria to be a valid input to is_stable.
```

What's happening is that `ParticleList("")` creates an empty `ParticleList` instead of giving an `InvalidParticleError`.  This is because the empty string is being treated as an iterable with nothing in it.  So, this is something that'll need to be addressed when we address #1436.